README.md

```diff
@@ -20,9 +20,9 @@ limitations under the License.
 
 **LATEST RELEASE: You are currently on the main branch which tracks
 under-development progress towards the next release. The latest
-release of the Triton Model Analyzer is 1.22.0 and is available on
+release of the Triton Model Analyzer is 1.24.0 and is available on
```

Comment:
is this correct? Was the last version not incremented properly?

Reply:
It looks like it wasn't there why I made this substitution here.